### ISSUE TITLE
Mini Cart: it's not possible to navigate to the list of products with the keyboard

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#5575]
## Describe the bug
Mini Cart drawer contents are not accessible using the keyboard. Heads-up that in the past they were, so at some point there has been a regression.

## To reproduce
1. Add the Mini Cart block to a post or page.
2. Pressing <kbd>Tab</kbd>, focus on the Mini Cart block and click it (<kbd>Space</kbd>).
3. When the Mini Cart drawer opens, use the <kbd>Tab</kbd> key to navigate through the projects.
4. Notice it's not possible, for some reason the focus keeps doesn't go inside the list of products, it only focus the close button and the list wrapper.

## Expected behavior
Using <kbd>Tab</kbd>, I should be able to focus the Product title, quantity selector, etc.

## Screenshots
https://user-images.githubusercontent.com/3616980/149785892-839ea673-c9c0-49ef-b627-2326e9633af2.mp4

### WordPress (please complete the following information):
* Core version: 5.8.1
* Gutenberg version (if installed): 12.3.2
* WooCommerce version: 6.1.0
* WooCommerce Blocks version: `trunk`

**Comment by dinhtungdu** [comment]
@Aljullu I can't reproduce the issue. I can navigate through the products table using the keyboard only. Tested with both normal theme (Storefront) and block theme (Twenty Twenty-Two). Can you please pull the latest commit of this repo and test it again? I tested both WC 6.1.0 and WC 6.3.0, WP 5.8.1 and WP 5.9.

https://user-images.githubusercontent.com/5423135/151330439-a094416c-9e75-42f9-8c7f-003b7a9a7cd6.mov

**Comment by Aljullu** [comment]
Oh, right, seems to be working alright now. Not sure what was going on when I tested. :confused:
